samples/holiday-package-sample/services/samples/holiday/HolidayPackageDetails.bal

```diff
@@ -0,0 +1,189 @@
+package samples.holiday;
```

Comment:
It doesn't make sense to have a single bal file/service for all the backend services, Please have different services for different entities. We can't even have a single service for same category. All must of individual services. Please think about the real word scenario and mock those services. 
If not it is hard for someone to understand this.